CMSgov/design-system
# [NO-TICKET] Export the healthcare version of ThirdPartyExternalLink

## Summary

- The healthcare version of `ThirdPartyExternalLink` contains a default `learnMoreUrl` which currently isn't being exported.

## How to test

1. run `yarn clear && yarn install && yarn build`
2. Copy the `packages/ds-healthcare-gov/dist` directory to your project and overwrite the `node_modules/@cmsgov/ds-healthcare-gov` with it
3. Verify that the learn more url in the `ThirdPartyExternalLink` modal popup directs the user to `https://www.healthcare.gov/using-this-site/`
